### PR TITLE
Plesae check folder Phani-1 for files

### PR DESCRIPTION
**Pull request by kulkarni90**
Check the files and comment if you have any.

### CODE REVIEW

File: PHANI-1/Git-Command.txt [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+   37.git remote  -> Mangae set of tracked repositories.
+
+   38.git-help    -> Display information about Git.
+
+   39.git -shell  -> Restricted login shell for Git-only SSH access.
+
+   40.git log     -> Lists version history for the current branch.  
+
+   41.git log --follow -> Lists version history for a file,including renames.
+
```
Comment (david1412) [inline PHANI-1/Git-Command.txt:52]: Remove the spacing. Please done repeat these commands in next PR. Looks good

File: PHANI-1/Scrum.txt [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+             * Move from idea to delivery more quickly
+             * Drive higher customer satisfaction
+             * Increase employee morale
+
+
+   Scrum defines three roles
+
+   1.PRODUCT OWNER : 
+    
+               The Product Owner defines the what--as in what the product will look like and what features it should contain.
```
Comment (david1412) [inline PHANI-1/Scrum.txt]: Indent should be properly done.

Example: 
your text:

The Product Owner is expected to incorporate stakeholder feedback to create the highest value product increments each and every sprint. 
	       Product Owners maintain the product backlog and ensures that everyone knows the priorities.

I want : 
The Product Owner is expected to incorporate stakeholder feedback to create the highest value product increments each and every sprint. 
Product Owners maintain the product backlog and ensure that everyone knows the priorities.

**Review by pradeepm2406 — COMMENTED** [review]
1. The requirement of the task is to have 10 points per topic in each branch. Please select only 10 points in all three files.
2. The formatting in the files is not good. please check the line spacing in all three files.

File: PHANI-1/Scrum.txt [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            * Coach: Facilitates meetings, conversations, and improvements.
+            * Protector: Runs interference so the team can remain focused.
+            * Servant Leader. Leads without authority and puts the team first. 
+            * Agile Advocate: Reinforces agile principles throughout the organization
+
+
+   3. DEVELOPMENT TEAM :
+       
+             The Development Team decides how to accomplish the work set forth by the Product Owner. Development Teams are structured and empowered to organize and 
+	     manage their own work. The resulting synergy optimizes overall efficiency and effectiveness.
```
Comment (david1412) [inline PHANI-1/Scrum.txt]: Indent spacing please.

File: PHANI-1/Scrum.txt [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+   3. DEVELOPMENT TEAM :
+       
+             The Development Team decides how to accomplish the work set forth by the Product Owner. Development Teams are structured and empowered to organize and 
+	     manage their own work. The resulting synergy optimizes overall efficiency and effectiveness. 
+
+
+        Development teams have the following characteristics :
+
+            * They are self-organizing. No one (not even the ScrumMaster) tells the development team how to turn Product Backlog into Increments of potentially releasable functionality;
```
Comment (david1412) [inline PHANI-1/Scrum.txt]: Indent spacing

File: PHANI-1/Scrum.txt [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+            * Development teams are cross-functional, with all the skills as a team necessary to create a product Increment;
+            * Scrum recognizes no titles for development team members, regardless of the work being performed by the person;
+            * Scrum recognizes no sub-teams in the development team, regardless of domains that need to be addressed like testing, architecture, operations, or business analysis.
```
Comment (david1412) [inline PHANI-1/Scrum.txt]: Indent spacing

File: PHANI-1/Scrum.txt [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+	    * Individual development team members may have specialized skills and areas of focus, but accountability belongs to the development team as a whole.
+
+        Development Team Size :
+               
+	       The ideal size for a development is between 3 and 9 people, not including the ScrumMaster and product owner. Any smaller and the team couldn’t accomplish enough each sprint. 
+	       Any larger and communication becomes complex and cumbersome.
+
```
Comment (david1412) [inline PHANI-1/Scrum.txt]: Remove the spaces here.

File: PHANI-1/Scrum.txt [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+	      Prescribed events are created by Product Manger to keep reqularity and to minimize the need for meetings.
+
+	      It consists of
+
+		      1.SPRINT : Scrum Teams work in short timeframes called sprints. Sprints can be as short as one week or as long as one month. 
+			         Sprints happen one right after the other, with no breaks, to maintain a steady project cadence.  
+                                 Each sprint begins with a plan and ends with a review of the work completed and an additional review of the way in which the team worked together.
+		      
+		      2.SPRINT PLANNING : The entire Scrum Team collaborates and discusses the desired high-priority work for the Sprint and defines the Sprint Goal.
```
Comment (david1412) [inline PHANI-1/Scrum.txt]: Indent spacing, please correct it

**Review by david1412 — COMMENTED** [review]
Explanation looks good. 
Please remove spaces and use indent. See the comments and work further.

**Review by prudhvidhargudimalla — COMMENTED** [review]
used more spacing

File: PHANI-1/Git-Command.txt (hunk 1 above)
Comment (kulkarni90) [reply PHANI-1/Git-Command.txt:52]: I purposely gave space at the beginning of topic /lines to gave a paragraph/border look 

surely will correct should i do  it in this PR or in next PR

File: PHANI-1/Scrum.txt (hunk 2 above)
Comment (kulkarni90) [reply PHANI-1/Scrum.txt]: I purposely gave space at the beginning of topic /lines to gave a paragraph/border look

surely wiIl update

**Review by kulkarni90 — COMMENTED** [review]
I purposely gave space at the beginning of topic /lines to gave a paragraph/border look

surely will update

**Review by nithindeva — COMMENTED** [review]
Hi phani you gave some useful information, but i have a doubt did you copy paste it or typed it. and the  text file is too lengthy please concise it to limited points.

**Review by kulkarni90 — COMMENTED** [review]
Indentation and spacing are corrected.

**Comment by kulkarni90** [comment]
> 1. The requirement of the task is to have 10 points per topic in each branch. Please select only 10 points in all three files.
> 2. The formatting in the files is not good. please check the line spacing in all three files.

1.i Thought to keep all data at one point relating to one topic
2. ok anna surely will  make changes regarding formatting .

File: PHANI-1/Git-Command.txt (hunk 1 above)
Comment (nithindeva) [reply PHANI-1/Git-Command.txt:52]: It is too big file to review at one go why don't you limit it to 10 points

**Review by nithindeva — COMMENTED** [review]
You added too much info in one file its hard to review

**Comment by kulkarni90** [comment]
> Hi phani you gave some useful information, but i have a doubt did you copy paste it or typed it. and the text file is too lengthy please concise it to limited points.

Hi anna thanks . to be frank 75% typed, 25% copied  data source : google, print material, class notes-- made a read and edit it to my understating.

**Review by nithindeva — COMMENTED** [review]
why don't you use numbering instead of spaces.

**Review by pradeepm2406 — COMMENTED** [review]
Looks ok, but you may not have new information to add in the next set of tasks. Other than that it is fine for me.

**Review by david1412 — COMMENTED** [review]
Git commands and Linux commands. Then the PR will be approved!

**Review by david1412 — APPROVED** [review]
Looks Good.

**Review by pradeepm2406 — APPROVED** [review]
All good

**Review by nithindeva — APPROVED** [review]

**Pull request merged**